### ISSUE TITLE
request for height setting fog.

### ISSUE THREAD

**Issue by earthling45** [TheGameCreators/GameGuruRepo#2995]
![kubNR4t](https://user-images.githubusercontent.com/35744254/165863908-daddd49c-c091-43ee-b62f-fc21ba19398a.jpeg)
![h0kNfB4](https://user-images.githubusercontent.com/35744254/165863933-18f8ddda-45bd-4b80-8bde-875e3192f35a.jpeg)
![MZvyK8l](https://user-images.githubusercontent.com/35744254/165864009-f0e8fe36-26e7-454b-a6c2-7b6a1b5760c5.jpeg)

Hi, my request for GGM is to have the possibility to also adjust the height of fog so that we could have groundfog as shown in the pictures above. (hopefully they are visible) :)
I think certainly in a forest scene this will look fantastic.

**Comment by ArgentArts** [comment]
I was JUST going to come here and request ground fog! I'd love to have some means of creating fog with a height limitation! Thanks for bringing this up!

**Comment by Kitakazi** [comment]
Could be a great particle effect. Ive seen particle effects used to get similar ground level spooky fog in other engines.

**Comment by MonkeyFrogStudio** [comment]
I've made a few particle effects for this, but they can also hit the engine a bit hard if you need a lot of them for several areas. Which is why it would be great to have it as an engine feature. :)

**Comment by earthling45** [comment]
Indeed, controlling the height of fog would not affect performance but we do get that ground fog effect. :)

I also think that this is more realistic than particles because those particles are placed near for view and do not cover the whole ground in the distance like ground fog usually does.
Moreover, if we have these controls with values it should easily be possible to bring it alive (dynamic) in a game.
With that i mean, either it becomes misty or ground fog creeping in or it is subsiding during gameplay.
It could be an element of dynamic weather when that is implemented.

**Comment by MonkeyFrogStudio** [comment]
That, and it would be simple, for example, to cover an entire level (especially an interior level) with the click of a button (or a few settings) than having to place dozens of particles all over the place. ;)

**Comment by Necrym59** [comment]
i wonder if the cloud system could be adapted for low level fog?
